ReactNativeClient/lib/components/action-button.js

```diff
@@ -93,7 +93,7 @@ class ActionButtonComponent extends React.Component {
 		}
 
 		if (!buttonComps.length && !this.props.mainButton) {
-			return <ReactNativeActionButton style={{ display: 'none' }} />;
+			return null;
```

Comment:
There was probably a reason why it was set to `display: none` and not just `null`. Does it still work if you start from an empty app, and add a notebook. Then does the Action button show up?